app/api/clima/page.js

```diff
@@ -0,0 +1,10 @@
+async function fetchData() {
+    const res = await fetch('https://api.openweathermap.org/data/2.5/weather?q=Mar%20del%20Plata,AR&appid=a3f3933bf95b00578ff3adc1d8402640')
+    
```

Comment:
Falta retornar el response:

```suggestion
        if (response.ok) {
       const data = await res.json();
       return data;
     } else {
       throw new Error('Error en encontrar data');
     }
```